File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.go
func twoSum(numbers []int, target int) []int {
    l := 0
    h := len(numbers) -1
    res := make([]int, 2)
    for l<h{
        if numbers[l] + numbers[h] > target{
            h = h-1
        }else if numbers[l] + numbers[h] < target{
            l = l+1
        }else{
            res[0], res[1] = l+1, h+1
            return res
        }
    }
    return res
}

File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        map = {}
        for i in range(len(numbers)):
            if map.get(numbers[i]) == None:
                map[target-numbers[i]] = i
            else:
                return [map[numbers[i]]+1, i+1]
        return [-1,-1]
            